show/kube.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python -u
+# -*- coding: utf-8 -*-
+
+import os
+
+import click
+
```

Comment:
Remove this blank line. `click`, `sonic_py_common` and `utilities_common` are all technically "third-party" libraries relative to `show`.

Reply:
Done